app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -27,6 +27,20 @@ const EMOJI_LIST_OFFSET_LEFT = 8;
 const EMOJI_LIST_OFFSET_TOP = 8;
 const QUERY_MIN_LENGTH = 1;
 
+// DO NOT USE COLON WITH LOWERCASE LETTERS IN THE KEYS, or you will prevent searching emojis.
+// For example, while :D should be replaced with unicode symbol, :d should allow searching for :dancer:
+const EMOJI_INLINE_REPLACEMENT = {
```

Comment:
@mikeger I was waiting for this comment, I will update :+1: 

I have one question... I just have to ask 😄: 

Is it set in stone that `:)` transforms into 😊 and not 🙂 ? The second one feels like a better match, so I wonder if it is possible to reconsider and also update iOS client to transform `:)` into 🙂?

The other replacements are fine, except that I will remove a few, those that have lowercase letter after colon (the reason is described in the code comment just above).